src/core/EspressoSystemStandAlone.cpp

```diff
@@ -43,9 +41,6 @@ EspressoSystemStandAlone::EspressoSystemStandAlone(int argc, char **argv) {
   Communication::init(mpi_env);
 
   // default-construct global state of the system
-#ifdef VIRTUAL_SITES
-  set_virtual_sites(std::make_shared<VirtualSitesOff>());
-#endif
```

Comment:
The comment on line 43 is now referring to nothing and will cause confusion, since `mpi_loop()` on line 46 doesn't default construct anything.